### ISSUE TITLE
Django 1.9 incompatibility.

### ISSUE THREAD

**Issue by holms** [hovel/imagestore#61]
When launching project this happens. For now it's Django 1.8 this will work, but in Django 1.9 this will fail to load. 

```
/home/djangocms/djangocms/local/lib/python2.7/site-packages/imagestore/utils.py:8: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
from django.utils.importlib import import_module
```

**Comment by lampslave** [comment]
I'm plan to add support of 1.9 after next release of django CMS, because django CMS is required for tests.

**Comment by marc-gist** [comment]
:+1:

**Comment by lampslave** [comment]
Closed via 3269ebdb477a76b41b4e7e85612701c6d5490f12.
